site/content/en/docs/manual/advanced/consensus.md

```diff
@@ -0,0 +1,116 @@
+
+| | |
+|---|---|
+|![basket_hound_ear_covered](/images/basket_hound_ear_covered.jpeg)|![basket_hound_ear_not_covered](/images/basket_hound_ear_not_covered.jpeg)|
+
+
+See:
+
+- [Use Cases](#use-cases)
+- [Terminology:](#terminology)
+- [Working](#working)
+- [Finding Overlap between two annotations:](#finding-overlap-between-two-annotations)
+- [Step by Step Guide](#step-by-step-guide)
+
+## Use Cases
+
+The basic use case is annotating the entire dataset with multiple annotators to obtain high-quality annotations in the dataset. In some cases it may work, but typically, however, this way of annotating is prohibitively expensive, as you need to annotate everything several times. There are several ways how the situation can be improved.
+
+1. You can use several highly skilled annotators and rely on their consensus to create high-quality Ground Truth annotations. Typically, the Ground Truth subset contains a small portion of the whole dataset (e.g. 2-5%), so it's relatively easy to annotate it several times. These annotations are used then to validate work of different annotators randomly, so it's very desirable to avoid single expert bias in Ground Truth, and this is one of the key benefits of the consensus annotation. This method requires extra work, but it's much lower than annotating all the dataset several times. For instance, annotating a 3% of images 5 times will result in extra 3% * (5 - 1) = 12% overhead, comparing to single GT annotations.
+2. In cases of points and poly-line / poly-shape annotations, annotators can add more annotations without a fixed structure, which can introduce significant variation. Similarly, for skeleton annotations, while there is a structure, annotators can move the points around. Due to the presence of numerous points, this can lead to significant variations in annotations for the same data. Using a consensus-based approach for such data types can help reduce these variations, which may arise from minor errors in placing the points.
+   1. For example, in the figure, an annotator might place a point representing the foot anywhere across the pink region, leading to significant variation in the annotations. ![pink_shoes](/images/pink_shoes.jpeg)
+
+
+## Terminology:
+1. `min_iou_threshold`: This is the same as `iou_threshold` param in Quality Settings. It's used for distinction between matched / unmatched annotations. A annotation is considered in the consensus only with the annotations it has an overlap greater than or equal to this
+2. `agreement_score_threshold`: After consensus the merged annotation, is assigned a score based on it's overlap with other annotations in the cluster, (add label score thing also). Merged annotation with this score lesser than `agreement_score_threshold` are discarded.
```

Comment:
Will you add `(add label score thing also)`?

Reply:
Should I add it as here in a cluster all the annotations will be of the same label?

Reply:
Just make the resulting values understandable to the users. If you're sure the label score is always 1, it can be removed.